src/QmlControls/QGCSwitch.qml

```diff
@@ -9,20 +9,43 @@
 
 import QtQuick
 import QtQuick.Controls
-import QtQuick.Controls
 
 import QGroundControl.Palette
 import QGroundControl.ScreenTools
 
 Switch {
-    id: _root
-    QGCPalette { id:qgcPal; colorGroupEnabled: true }
+    id: control
+
+    readonly property int _radius: 3
+
+    QGCPalette {
+        id:                 qgcPal
+        colorGroupEnabled:  true
+    }
+
+    contentItem: Text {
```

Comment:
This should just be QGCLabel. That way it will automatically follow any styling changes.

Reply:
Yes, changed